x/rollapp/keeper/rollapp.go

```diff
@@ -58,8 +58,6 @@ func (k Keeper) CheckAndUpdateRollappFields(ctx sdk.Context, update *types.MsgUp
 
 		if update.GenesisInfo.GenesisAccounts != nil {
 			current.GenesisInfo.GenesisAccounts = update.GenesisInfo.GenesisAccounts
-		} else if current.GenesisInfo.GenesisAccounts != nil {
-			current.GenesisInfo.GenesisAccounts.Accounts = nil
```

Comment:
this was added to support nullifying genesis accounts when providing an empty list. is this still supported now that it's removed? if it is, didn't see how.

Reply:
For anyone else reading: as discussed on slack

Frontend will always pass up the updated value, be it nil, or some actual accounts.